### PR TITLE
feat(web): add formatDuration utility to format.ts

### PR DESCRIPTION
**Pull request by Pritom14**
## Summary

- Adds `formatDuration(ms: number): string` to `packages/web/src/lib/format.ts`
- Outputs compact human-readable durations: `"45s"`, `"1m 30s"`, `"2h 0m"`
- Returns `"0s"` for zero or negative input
- Appends 9 Vitest test cases to `src/lib/__tests__/format.test.ts` covering all rules and edge cases

## Test plan

- [x] `pnpm --filter @aoagents/ao-web test` — all 36 format tests pass
- [x] `pnpm --filter @aoagents/ao-web typecheck` — no errors

Closes #30

🤖 Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: packages/web/src/lib/format.ts [hunk 1]
```diff
@@ -114,6 +114,17 @@ export function getSessionTitle(session: DashboardSession): string {
   return session.status;
 }
 
+export function formatDuration(ms: number): string {
+  if (ms <= 0) return "0s";
```
[the remark this answers is not in the document]
Comment (Pritom14) [reply packages/web/src/lib/format.ts:132]: Fixed — added a JSDoc block documenting all output tiers and the sub-second truncation behaviour, matching the pattern of the other exports in the file.

File: packages/web/src/lib/__tests__/format.test.ts [hunk 2]
```diff
@@ -39,6 +39,38 @@ function makeSession(overrides?: Partial<DashboardSession>): DashboardSession {
...
+describe("formatDuration", () => {
+  it("returns '0s' for zero input", () => {
+    expect(formatDuration(0)).toBe("0s");
+  });
+
+  it("returns '0s' for negative input", () => {
+    expect(formatDuration(-1000)).toBe("0s");
+  });
+
+  it("shows seconds only for durations under 60s", () => {
```
[the remark this answers is not in the document]
Comment (Pritom14) [reply packages/web/src/lib/__tests__/format.test.ts:60]: Fixed — added a test case covering 1–999 ms inputs to document that they return `"0s"` via silent truncation.
